packages/engine/src/store/types.ts

```diff
@@ -19,52 +19,52 @@ export interface NodeBase {
   key: string
 }
 
-export enum NodeTypes {
-  Number = 'number',
-  Boolean = 'boolean',
-  String = 'string',
-  Enum = 'enum',
-  Vector3 = 'vector3',
-  RGB = 'rgb',
-}
```

Comment:
This TS enum has been removed, as it was causing some TS issues and we don't actually need it. We still have complete type safety because every type of node has the `valueType` associated with it.

This is also a necessary change for later, when we want users to be able to type their config files. With this enum, they'd need to import `NodeTypes` just to give each param in their config a `valueType`.